portal/views/teacher/teach_new.py

```diff
@@ -68,8 +68,56 @@
 from portal.templatetags.app_tags import cloud_storage
 
 
-@login_required(login_url=reverse_lazy('home_new'))
-@user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('home_new'))
+@login_required(login_url=reverse_lazy('login_new'))
+@user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('login_new'))
+def materials_viewer_new(request, pdf_name):
+
+    def _getLinks():
+        links = PDF_DATA[pdf_name]['links']
+        link_titles = []
+        for link in links:
+            link = link.replace('_', ' ').title()
+
+            if (link[0] == 'K') | (link[1] == 'k'):
```

Comment:
that's going to be funny to localise if we get non-uk teaching materials... ;) Maybe we should have a white list of words to capitalise like KS, LKS... and use startswith() for example